frameworks/person-escort-record/manifests/risk-information.yml

```diff
@@ -67,4 +67,5 @@ steps:
     slug: other-risks
     questions:
       - communication-needs
+      - religious-cultural-needs
```

Comment:
I've just noticed we already have [this question](https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/blob/main/frameworks/youth-risk-assessment/questions/religious-or-cultural-needs.yml) in the youth risk assessment framework. 

Maybe we should re-use this question and can adapt the hint text in both to match this if we think it needs updating. It would be better to ask this question consistently in my opinion. 

And that could also open the question of whether we then need it in both seeing as a youth risk assessment is currently always asked before the Person Escort Record.

cc/ @ElizabethGoddard

Reply:
Good point... if we're going to ask it now anyway in the Person Escort Record, definitely sounds like we should remove it from the Youth Risk Assessment, but I'll let @ElizabethGoddard take a look!

Reply:
Hi @teneightfive and @thomasleese just had confirmation from Adam Luck that it should be fine to update the question in the YRA to match. As mentioned, we will address the duplication issue along with other YRA issues at a later date. Thanks!

Reply:
Cool, thanks, I've made that change in 2d9c252!